blackbox/docs/sql/general/constraints.rst

```diff
@@ -76,3 +76,17 @@ non-null values.
 
 The columns that are part of the primary key of a table are NOT NULL by
 default.
+
+.. _primary_key_column_constraint:
+
+``PRIMARY KEY``
+---------------
+
+The PRIMARY KEY column constraint specifies that a column of a table can
```

Comment:
I think this is a bit missleading. If a column has a `PRIMARY KEY` constraint it simply declares that this column is part of the tables primary key. There is no semantic separation between "column primary key" and "table primary key".  The former doesn't exist really.